Program.cs

```diff
@@ -22,37 +25,47 @@
 while (keepAlive)
 {
 MainLoop:
-	UpdateWindowPosition(window);
-	await Task.Delay(100);
-
-	if (process.HasExited)
-	{
-		var oldProcessId = process.Id;
-		var retryCount = 0;
-		while (retryCount < 2)
-		{
-			var matchingProcesses = Process.GetProcessesByName("BF1942");
-			if (matchingProcesses.Length == 1)
-			{
-				process = matchingProcesses[0];
-				Console.WriteLine($"Process has changed ({oldProcessId} -> {process.Id})");
-				window = await process.WaitForMainWindowAsync();
-				window.RemoveBorders();
-				goto MainLoop;
-			}
-			retryCount++;
-			await Task.Delay(TimeSpan.FromSeconds(1));
-		}
-		keepAlive = false;
-		Console.WriteLine($"Process has exited ({oldProcessId})");
-	}
+
+    UpdateWindowPosition(window);
+    await Task.Delay(100);
+
+    if (process.HasExited)
+    {
+        var oldProcessId = process.Id;
+        var retryCount = 0;
+
+        while (retryCount < 2)
+        {
+            var matchingProcesses = Process.GetProcessesByName("BF1942");
+            if (matchingProcesses.Length == 1)
+            {
+                process = matchingProcesses[0];
+
+                Console.WriteLine($"[Borderless1942]: [{DateTime.Now:MM/dd/yyyy - hh:mm:ss tt}] [BF1942 Process Has Changed] [Map Change Detected] ({oldProcessId} -> {process.Id})");
```

Comment:
While this definitely does happen on map change, it isn't the only reason it happens. Just navigating from in-game to the main menu switches process (which is just weird in my opinion but here we are). It is partly why I was ambiguous about the cause of the process change.

So besides my comment about date format, I'd probably just say "Process Change Detected" instead of specifically "Map Change Detected".

Reply:
I was not aware that the process also changed when navigating from in-game to the main menu.  Although now that you mention that, I do seem to remember that you mentioned that fact in your blog post.  I will remove the map change detected text from the console message.